doc/concepts/concepts.rst

```diff
@@ -1,13 +1,228 @@
+========
 Concepts
 ========
 
-These are the **why-docs** that discuss the design of MoveIt and this website.
+The following list major MoveIt concepts and components and describes how they tie in with each other. For tutorials, see the `tutorials <https://moveit.picknik.ai/>`_ page.
+
+.. image:: /_static/images/moveit_pipeline.png
+
+***********************
+The ``move_group`` node
+***********************
+
+The figure above shows the high-level system architecture for the primary node provided by MoveIt called [move_group](http://docs.ros.org/noetic/api/moveit_ros_move_group/html/move__group_8cpp.html). This node serves as an integrator: pulling all the individual components together to provide a set of ROS actions and services for users to use.
+
+.. image:: /_static/images/move_group.png
+
+User Interface
+--------------
+
+The users can access the actions and services provided by ``move_group`` in three ways:
+
+- **In C++** - using the `move_group_interface <http://docs.ros.org/noetic/api/moveit_ros_planning_interface/html/classmoveit_1_1planning__interface_1_1MoveGroupInterface.html>`_ package that provides an easy to setup C++ interface to move_group
+
+- **In Python** - using the `moveit_commander <http://docs.ros.org/noetic/api/moveit_commander/html/classmoveit__commander_1_1move__group_1_1MoveGroupCommander.html>`_ package
+
+- **Through a GUI** - using the `Motion Planning plugin to Rviz <https://moveit.picknik.ai/foxy/doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>`_ (the ROS visualizer)
```

Comment:
```suggestion
- **Through a GUI** - using the :doc:`Motion Planning plugin to Rviz </doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.html>` (the ROS visualizer)
```

Reply:
The suggested change doesn't seem to work... I'm trying to figure out how to do relative links in RST if you they are supported.

Reply:
Apparently like this: 
```doc:`Motion Planning plugin to Rviz <../tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial>` ```